src/containers/DataTable/index.js

```diff
@@ -70,14 +71,32 @@ export function DataTable ({
 							<i className={title.icon}></i> : title.icon
 						}
 						{title.text}
+						{button && 
```

Comment:
@titkov49 What if I would like to have a dropdown here or something else? Is it possible to create the component customizable?

Reply:
I think It would be better to create custom dropdown instead

Reply:
Yes, I think we would use that. If the dropdown can be customized (e.g. if I can modify the props of dropdown items), it would be great